core/loadpoint.go

```diff
@@ -1062,8 +1062,8 @@ func (lp *Loadpoint) fastCharging() error {
 	return err
 }
 
-// pvScalePhases switches phases if necessary and returns if switch occurred
-func (lp *Loadpoint) pvScalePhases(sitePower, minCurrent, maxCurrent float64) bool {
+// pvScalePhases switches phases if necessary and returns if switch occurred and new target current
+func (lp *Loadpoint) pvScalePhases(sitePower, minCurrent, maxCurrent float64) (bool, float64) {
```

Comment:
Aktuell wird der Rückgabewert aus `pvScalePhases` ja ignoriert. Den könnten wir wieder verwenden. Mal ein ganz einfacher Vorschlag: 

    return true

nur wenn 1p -> 3p. In dem Fall dann in `pvMaxCurrent`:

    maxCurrent = minCurrent

um es auf diesen einen Wert einzugrenzen. Der Rest von `pvMaxCurrent` sollte einfach ohne Änderung durch laufen.

Würde das nicht reichen oder geht dann was anderes kaputt?

Reply:
Falls wir den Rückgabewert für die Tests brauchen könnte man auch 0 (keine Änderung) oder 1 (runter) und 3 (hoch) zurück geben. Dann müssten wir auf 3 reagieren.

Reply:
Ohne jetzt einen Streit vom Zaun brechen zu wollen:
Ich habe jetzt noch genau eine Änderung gemacht um sicherzustellen, dass maxCurrent nicht ansteigen kann.
Das ist die letzte Änderung, die ich an diesem PR vornehmen werde. 
Warum die gewählte Lösung in vielen Fällen bessere Ergebnisse erzielen sollte als eine einfache Limitierung auf minCurrent habe ich bereits in der Dikussion mit premultiply erläutert.
Ich habe das aus Gefälligkeit marioth1 gegenüber implementiert, mir ist das auftretende Verhalten ehrlich gesagt komplett egal und ich werde keine Zeit mehr investieren um diesen PR gegenüber irgendwem zu verteidigen.

Reply:
Wie gehabt. Diese Diskussion bin ich nicht bereit zu führen.